test/message/test_runner_output.js

```diff
@@ -383,3 +383,9 @@ test('unfinished test with unhandledRejection', async () => {
     setTimeout(() => Promise.reject(new Error('bar')));
   });
 });
+
+test('should not omit top-level diagnostics', () => {
```

Comment:
this test is redundant, such a test already exists here: https://github.com/nodejs/node/blob/de70efa97fc59722a0e61f285828643a6efca3cf/test/message/test_runner_output.js#L80-L84

I think to avoid merge conflicts it is preferable to remove this test